projects/vdk-control-cli/src/vdk/internal/control/configuration/vdk_config.py

```diff
@@ -208,7 +209,12 @@ def read_configuration(self, section, option, fallback=None):
             log.debug(f"Reading configuration file: {configuration_file_path} ...")
 
             config_parser = ConfigParser()
-            config_parser.read(configuration_file_path)
+
+            read_job_configuration(
```

Comment:
Here we are not reading job configuration . 

This file is read for any VDK CLI command - not those related to job configuration. If that was so we would have used JobConfig class. 

Maybe name the method read_ini_config_file ? or something like this

Reply:
Renamed, thanks